imports/logger/server.lua

```diff
@@ -115,7 +115,8 @@ end
 
 if service == 'loki' then
     local lokiUser = GetConvar('loki:user', '')
-    local lokiPassword = GetConvar('loki:password', '')
+    local lokiKey = GetConvar('loki:key', '')
+    local lokiPassword = GetConvar('loki:password', lokiKey ~= '' and lokiKey or '')
```

Comment:
uhh
```lua
local lokiPassword = GetConvar('loki:password', GetConvar('loki:key', ''))
```
?

Reply:
Yea, fair enough, overthough it somehow myself.